MapboxDirections/MBRouteStep.swift

```diff
@@ -428,14 +428,14 @@ public class RouteStep: NSObject, NSSecureCoding {
     // MARK: Getting Details About the Maneuver
     
     /**
-     A string with instructions in English explaining how to perform the step’s maneuver.
+     A string with instructions explaining how to perform the step’s maneuver.
      
      You can display this string or read it aloud to the user. The string does not include the distance to or from the maneuver. If you need to localize or otherwise customize the instructions, you can construct the instructions yourself using the step’s other properties.
      */
-    public let instructions: String
+    public let instructions: String?
     
     public override var description: String {
-        return instructions
+        return instructions ?? maneuverType?.description ?? maneuverDirection?.description ?? "step"
```

Comment:
It isn’t supposed to be, but we do use it as sort of a back-to-JSON thing in this library.